test/e2e/autoscale_test.go

```diff
@@ -72,20 +73,16 @@ func generateTrafficBurst(clients *test.Clients, logger *logging.BaseLogger, wan
 				} else {
 					logger.Errorf("Nil response. No response headers to show.")
 				}
+				return err
```

Comment:
Does it make sense to drop this line and `return err` below? It should have the same value.

Reply:
that's correct. It depends how explicit you want to be. 

If  I have handle the error in any way, then I only return the err explicitly when it cannot be nil.

if err != nil {
 // Logic
 return err
}
return nil

It makes the logic more understandable at first sight.